cmd/cu/diff.go

```diff
@@ -8,17 +8,21 @@ import (
 )
 
 var diffCmd = &cobra.Command{
-	Use:               "diff <env>",
-	Short:             "Show what files an agent changed",
+	Use:   "diff <env>",
+	Short: "Show what files an agent changed",
 	Long: `Display the code changes made by an agent in an environment.
-Shows a git diff between the environment's state and your current branch.`,
+Shows a git diff of all changes made since the environment was created.
+Use -b to compare against a specific branch instead of showing full diff.`,
 	Args:              cobra.ExactArgs(1),
 	ValidArgsFunction: suggestEnvironments,
-	Example: `# See what changes the agent made
+	Example: `# See what changes the agent made (full diff)
 cu diff fancy-mallard
 
+# Compare against main branch
+cu diff fancy-mallard -b main
+
 # Quick assessment before merging
-cu diff backend-api`,
+cu diff backend-api -b main`,
```

Comment:
in raw git, isn't this equivalent to `git diff main fancy-mallard`? or `git diff ancy-mallard main` we should make sure we're copying that UX wherever possible.

also why does it say backend-api, those are adverb animals lol

Reply:
Any opinion from a git UX POV?